src/index.js

```diff
@@ -20,9 +19,21 @@ module.exports = ({
   const staticPublic = serveStatic(join(__dirname, '../public/'));
   const indexP = readFileP(indexPath, {encoding: 'UTF8'});
 
+  const injectHeader = (req, res, next) => {
+    res.setHeader('Swagger-API-Docs-URL', apiDocs);
+    next();
+  };
+
+  const serveIndex = (req, res, next) => {
+    if (req.path !== '/') return next();
+
+    return indexP.then(file => res.send(file)).catch(next);
+  };
+
   router.use((req, res, next) => {
     if (req.path === swaggerUi) {
-      return res.redirect(`${swaggerUi}/${parse(req.url).search || ''}`);
+      const {search} = new URL(req.url, 'http://this-is-a-stub-base.fix');
```

Comment:
`http://this-is-a-stub-base.fix` ?

Reply:
bizarre comme base non?

Reply:
C'est une fausse base, facultative, sinon il explose :boom:
IMO l'interface [WHATWG URL](https://nodejs.org/api/url.html#url_the_whatwg_url_api) qui a quelques limitation.

Reply:
la comme on veut que le search, je peux me permettre de mettre une valeur stub.